docs/team.rst

```diff
@@ -21,216 +21,271 @@ folks.
    :backlinks: none
    :depth: 1
 
-.. _conference-teams:
+Terminology
+-----------
 
-Conference Teams
-----------------
-
-Each conference has a specific team that helps run that event. For this year,
-you can see who is helping out:
-
-* :doc:`Portland 2021 Team </conf/portland/2021/team>`
-* :doc:`Prague 2021 Team </conf/prague/2021/team>`
-* :doc:`Australia 2021 Team </conf/australia/2021/team>`
+Team
+  Group of humans responsible for a specific aspect of the community.
+
+Team Member
+  Human responsible for performing the team tasks. Reports to the team lead.
+
+Team Lead
+  Human responsible for managing the team. Accountable for the results of the team. Serves on the community council and votes on internal organizational changes.
+
+Community Council
+  Group of all the team leads. Proposes and votes on internal organizational changes, such as infrastructure and brand design.
+
+
+How teams operate and interact
+------------------------------
+
+- **All membership and leadership roles are staffed by enthusiastic opt-in.** We want to prioritize the well-being of our contributors, and we're happy to have you if you're happy to contribute to the community. 
+- Each team operates independently except for actions that impact other teams or the community.
+- Each team is responsible for implementing and monitoring community guidelines, including CoC, within their team's area. 
+- All teams leads are included in the community council, which can propose and vote on internal organizational changes and processes. 
+- Team leads can serve on multiple teams, as much as their capacity and well-being allows.
+- CoreOps team members act as interim team leads in case team leads step down without a replacement.
+- Teams are generally comprised of volunteers. Depending on the scope and time investment of team leads, payment can be offered for ongoing service to people who have shown dedication to their role in the community.
+
+Joining and leaving teams
+-------------------------
+
+Team members
+  - Team members serve for a minimum of three months.
+  - Membership renews automatically after joining. To step down from the team, members need to give a 14-day notice to the team lead before the end of the term.
+  - Team membership requirements:
+    - be an active member of the community for at least one year (online or at events)
```

Comment:
this sublist currently fails formatting and I can never remember how to do them right in rst